### PR TITLE
docs(attribute directives/TS/Dart): remove `Renderer`; improve text

### PR DESCRIPTION
**Pull request by wardbell**
@kwalrath - Please confirm that my Dart changes are correct. Then we should push ASAP so we can run with latest A2 bits.

### CODE REVIEW

File: public/docs/_examples/attribute-directives/dart/lib/highlight_directive_1.dart [hunk 1]
```diff
@@ -5,8 +5,7 @@ import 'package:angular2/angular2.dart';
 
 @Directive(selector: '[my-highlight]')
 class Highlight {
-  Highlight(ElementRef element, Renderer renderer) {
-    //el.nativeElement.style.backgroundColor = 'yellow';
-    renderer.setElementStyle(element, 'background-color', 'yellow');
+  Highlight(ElementRef element) {
+    el.nativeElement.style.backgroundColor = 'yellow';
```
Comment (kwalrath) [inline public/docs/_examples/attribute-directives/dart/lib/highlight_directive_1.dart]: el -> element, right?

**Comment by kwalrath** [comment]
LGTM. I'll confirm that this still runs once you commit this change.

**Comment by kwalrath** [comment]
What are the "latest" bits? You might need to modify pubspec.yaml to specify that version of Angular.

File: public/docs/_examples/attribute-directives/dart/lib/highlight_directive_1.dart (hunk 1 above)
Comment (wardbell) [inline public/docs/_examples/attribute-directives/dart/lib/highlight_directive_1.dart]: Argh! I'll fix and force push

**Comment by wardbell** [comment]
We don't have "latest bits" yet. We'll have to coordinate on the expected move to beta.2 which (if we are fortunate) will require no code changes.

**Pull request closed without merging**